tests/jax/integration/data_server_test_data.py

```diff
@@ -1,100 +0,0 @@
-# python3
```

Comment:
happy that we have removed this

Reply:
It was before attached to an old test was made (draft)